core/auth/src/main/java/software/amazon/awssdk/auth/credentials/ProcessCredentialsProvider.java

```diff
@@ -76,30 +76,19 @@ public final class ProcessCredentialsProvider
 
     private final String commandFromBuilder;
 
+    private final List<String> commandAsListOfStringsFromBuilder;
```

Comment:
Why are we not naming it as 
`commandsFromBuilder`

Reply:
Its still executed as a single command, the user is just specifying it in a list, so that it can be executed not-in-a-shell